config/config_baseline.yml

```diff
@@ -0,0 +1,27 @@
+data:
+  path_to_data: data
+  train_file: train_data_toy.parquet # this is for the train_test_split.py output
+  test_file: dev_data_toy.parquet # this is for the train_test_split.py and eval_f1.py output
+  train_file_name: data_gen_content_train_toy.json # this is the train file to be used by hugginface_ner.py
+  validation_file_name: data_gen_content_val_toy.json # this is the validation file to be used by hugginface_ner.py
+  test_file_name: data_gen_content_dev_toy.json # this is the test file to be used by hugginface_ner.py
+environment:
+  SEED: 0
+  val_size: 0.2 # this is used in span_prediction to split train dataset 
+bert: # bert model params, used in huggingface_ner.py
```

Comment:
we don't have `huggingface_ner.py` anymore, the comments need to be adjusted as well